code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1336,4 +1336,151 @@
 	. = 1
 
 
+/datum/reagent/medicine/burnmix
+	name = "BurnMix"
+	description = "Take in moderation. Heals damage at a snail's pace. Effects increase when used with Epinephrine or Perfluorodecalin. You don't want this in your body for longer than needed."
+	reagent_state = LIQUID
+	color = "#BD328A" //  R 189 G 50 B 138 , Medium Red Violet , Hue of Violet 
+	overdose_threshold = 12
+	taste_description = "dixel"
+	glass_name = "glass of ...red?"
+	glass_desc = "Looks like some chunky dark red mix."
+	reagent_weight = 4
+	addiction_threshold = 40
+	// Max healing non OD with best rolls + buffs: 0.2 + 0.15 + 0.2 = 0.55 
+	// Max healing with OD + best rolls and all buffs: 0.4 + 0.35 + 0.4 = 1.15
+
+/datum/reagent/medicine/burnmix/on_mob_life(mob/living/carbon/M)
+	var/heal_roll = pick(-0.1,-0.2) // Base heal, Non OD
+	var/slur = 10 // Slur 10 sets up for future slur gains as real effects seem to start at 30
+	var/jitter = 0
+	if(M.health <= M.crit_threshold)
+		heal_roll = 0 // Burnmix will not heal anyone inside of crit without epipen or morhpine
+	if(prob(10))
+		heal_roll = 0 // Burnmix has a chance not to basic heal at all
+	if(holder.has_reagent(/datum/reagent/medicine/epinephrine)) // start IF has Epinephrine in body
+		jitter = jitter + 5
+		slur = slur + 25
+		heal_roll = heal_roll - 0.15
+		if(prob(15))
+			M.losebreath++
+			if(prob(80))
+				M.adjustStaminaLoss(1.25*REM, 0) // end IF has Epinephrine in body
+
+	if(holder.has_reagent(/datum/reagent/medicine/perfluorodecalin)) // start IF has Perfluorodecalin in body
+		jitter = jitter + 5
+		slur = slur + 25
+		M.drowsyness += 0.5
+		heal_roll = heal_roll - 0.2 // start IF has Perfluorodecalin in body
+
+	if(prob(10)) // Start of base probability effects
+		var/tox_roll = pick(1,-1) // Rolls for damage or healing
+		if(tox_roll == 1)
+			M.emote("moan")
+		if(tox_roll == -1)
+			M.emote("twitch")
+		if(prob(90))
+			M.adjustToxLoss(tox_roll*REM, 0)
+		M.Jitter(jitter)
+	if(prob(1)) // Last set of non OD probability effects
+		M.Dizzy(jitter)
+		M.slurring = slur // End of base probability effects
+	if(heal_roll < 0) // Healing payload after calculations
+		if (M.stat == DEAD) // Non OD , if dead , 30% chance to discard healing payload
+			if(prob(70))
+				M.adjustFireLoss(heal_roll*REM, FALSE, FALSE, BODYPART_ORGANIC)
+				M.adjustBruteLoss(heal_roll*REM, 0)
+		else
+			M.adjustFireLoss(heal_roll*REM, 0) 
+			M.adjustBruteLoss(heal_roll*REM, 0) 
+	..()
+	. = 1
+
+/datum/reagent/drug/burnmix/overdose_start(mob/living/M)
+	to_chat(M, "<span class='userdanger'>You feel the BurnMix increase in potency.</span>")
+
+
+/datum/reagent/medicine/burnmix/overdose_process(mob/living/M)
+	var/heal_roll = pick(-0.2,-0.4)
+	var/slur = 20
+	var/jitter = 2
+	if(M.health <= M.crit_threshold)
+		heal_roll = heal_roll + 0.2 // Burnmix in overdose will heal inside of crit without epipen or perfluorodecalin. Being in crit means a roll for either no base healing or half reduction to base heal.
```

Comment:
```suggestion
		heal_roll -= 0.2 // Burnmix in overdose will heal inside of crit without epipen or perfluorodecalin. Being in crit means a roll for either no base healing or half reduction to base heal.
```

you subtract if you want to heal damage and you add if you want to deal damage

Reply:
Good catch Alex!